### ISSUE TITLE
Trainining with positive / negative pairs

### ISSUE THREAD

**Issue by paulaceccon** [tensorflow/similarity#185]
Is it possible to train with positive/negative pairs instead of using "more representative classes", i.e. following the same idea as here: https://keras.io/examples/vision/siamese_contrastive/? 
I don't want to know that two images are similar and represent the number 1, or a dog, for example. I wanna know that two images are similar with a score of x. More precisely, I have a data set of cars, and I want to know, given two images to the network, that they are similar with a score of x, but I don't wanna label each pair with a car id.

**Comment by owenvallis** [comment]
Closing this for now, but feel free to re-open if other questions come up.
